api/Document.json

```diff
@@ -8757,7 +8757,7 @@
               "version_added": false
             },
             "webview_android": {
-              "version_added": "37"
+              "version_added": "36"
```

Comment:
36 is not possible. I'm not going to mark every instance, but please change this throughout.

Reply:
These are the changes in #3884.  I did find something that references the existence of a WebView v36 in an Android beta channel, and historically this repository has kept browsers that remained in beta.  Alternatively, I’m happy to remove it if it’s more trouble than it’s worth!

Reply:
It refers to a developer prelease. That's not something you would find in the wild, so to speak.

Reply:
Similarly, Firefox Android 11 - 13 only had beta releases, so their compat data isn’t stored in this repository either.

Reply:
Alright.  Scrapping those changes!